src/util/cache/redis.ts

```diff
@@ -85,7 +85,7 @@ export class RedisCache implements Cache {
   public get(
     id: string,
     lookup: () => Promise<any>,
-    { ttl, cache }: { ttl: number, cache: boolean } = { cache: true, ttl: 86400 }
```

Comment:
@johnthepink @delianides pinging you guys on this because it will probably change our thresholds for now